neonvm/apis/neonvm/v1/virtualmachine_types.go

```diff
@@ -215,6 +221,46 @@ func (g Guest) ValidateForMemoryProvider(p MemoryProvider) error {
 	return nil
 }
 
+// LogicalTime allows to track progress of changes to a VM.
+type LogicalTime struct {
+	Value     int64       `json:"value"`
+	UpdatedAt metav1.Time `json:"updatedAt"`
```

Comment:
why `UpdatedAt` ? (asking because it seems odd that a logical timestamp cares about semantics like updates -- my naive assumption is that `LogicalTime` would have two components: "logical" (int) and "real" (metav1.Time), and it's up to the "application logic" (in this case, everyone else) to apply meaning on top of that)